src/packages/StorageBucketApi.ts

```diff
@@ -111,10 +121,12 @@ export default class StorageBucketApi {
    *
    * @param id A unique identifier for the bucket you are updating.
    * @param options.public The visibility of the bucket. Public buckets don't require an authorization token to download objects, but still require a valid token for all other operations.
+   * @param options.fileSizeLimit specifies the file size limit that this bucket can accept during upload
+   * @param options.allowedMimeTypes specifies the allowed mime types that this bucket can accept during upload
    */
   async updateBucket(
     id: string,
-    options: { public: boolean }
+    options: { public: boolean; fileSizeLimit?: number | string; allowedMimeTypes?: string[] }
```

Comment:
@fenos 
Here, when I used `fileSizeLimit: 20000000`, it threw me an error saying `the requested file_size_limit uses an invalid format, use 20GB / 20MB / 30KB / 3B`, but should the type of `fileSizeLimit` be just `string | null`?